include/deal.II/fe/fe_interface_values.h

```diff
@@ -1442,17 +1442,17 @@ class FEInterfaceValues
    *   is initialized with an hp::FECollection, an hp::QCollection, and possibly
    *   an hp::MappingCollection).
    */
-  template <typename CellIteratorType, typename CellNeighborIteratorType>
+  template <typename CellType>
   void
-  reinit(const CellIteratorType         &cell,
-         const unsigned int              face_no,
-         const unsigned int              sub_face_no,
-         const CellNeighborIteratorType &cell_neighbor,
-         const unsigned int              face_no_neighbor,
-         const unsigned int              sub_face_no_neighbor,
-         const unsigned int q_index       = numbers::invalid_unsigned_int,
-         const unsigned int mapping_index = numbers::invalid_unsigned_int,
-         const unsigned int fe_index      = numbers::invalid_unsigned_int);
+  reinit(const TriaIterator<CellType> &cell,
+         const unsigned int            face_no,
+         const unsigned int            sub_face_no,
+         const TriaIterator<CellType> &cell_neighbor,
```

Comment:
I don't understand this change. First, it shouldn't be `CellType` as template argument, but `AccessorType` (or perhaps `CellAccessorType`), but more importantly, I'm unclear what it is you are trying to enable with the change that wasn't possible before (or what you want to prevent that accidentally compiled before but shouldn't have)?

Reply:
> I'm unclear what it is you are trying to enable with the change that wasn't possible before (or what you want to prevent that accidentally compiled before but shouldn't have)?

The code evaluated the shape functions at the quadrature points AND loaded the DoF indices. You can do the latter only of `DoFCellAccessor`s.

Reply:
Ah, so you're saying that the change to the iterator type is only because you wanted to get access to the `Accessor` type of the iterator? You could have gotten that via `typename CellIteratorType::AccessorType`, which I think makes all of this easier to read because users don't typically know what classes are behind the cell iterators.